docs/source/methods/depletion.rst

```diff
@@ -257,3 +257,95 @@ choose one of two methods for estimating the heating rate, including:
 argument to the :class:`openmc.deplete.CoupledOperator` class.
+
+--------------
+Transfer Rates
+--------------
+
+OpenMC allows continuous removal or feed of isotopes by adding an
+extra transfer rate term to the depletion matrix. An application of this feature
+is the chemical processing of Molten Salt Reactors (MSRs), where one can
+model the removal of fission products or feeding fresh fuel into the system.
+
+A transfer rate as defined here is the rate at which nuclides are
+continuously removed/feed from/to a material.
+
+.. note::
+
+    A transfer rate can be positive or negative, indicating removal or feed
+    respectively.
+
+Mathematically, it can be thought as an additional term :math:`\mathbf{T}`
+in the depletion equation that is proportional to the nuclide density, which can be written as:
+
+.. math::
+
+  \begin{aligned}\frac{dN_i(t)}{dt} = &\underbrace{\sum\limits_j f_{j\rightarrow i}
+  \int_0^\infty dE  \; \sigma_j (E,t) \phi(E,t) N_j(t)  - \int_0^\infty dE \; \sigma_i(E,t)
+  \phi(E,t) N_i(t)}_\textbf{R} \\
+  &+ \underbrace{\sum_j \left [ \lambda_{j\rightarrow i} N_j(t) - \lambda_{i\rightarrow j} N_i(t) \right ]}_\textbf{D} \\
+  &- \underbrace{\epsilon_i t_i N_i(t)}_\textbf{T} \end{aligned}
+
+where the reaction term :math:`\mathbf{R}`, the decay term :math:`\mathbf{D}`
+and the new transfer term :math:`\mathbf{T}` have been grouped together so that
+:math:`\mathbf{A} = \mathbf{R}+\mathbf{D}-\mathbf{T}`.
+The transfer efficiency :math:`\epsilon_i` and the transfer rate coefficient
+:math:`t_i` define the continuous transfer of the nuclide :math:`i`, which
+behaves similar to radioactive decay.
+:math:`t_i` can also be defined as the reciprocal of a cycle time
+:math:`T_{cyc}`, intended as the time needed to process the whole inventory.
+
+For simplicity, :math:`\epsilon_i` and :math:`t_i`
+can be combined together in one single user-defined parameter that is defined
+again as :math:`t_i`.
+Thus, setting a transfer rate coefficient of 1 s\ :sup:`-1` at 100% efficiency, would be
+the same as setting 10 s\ :sup:`-1` at 10%.
+
+
+Note that this formulation assumes first order transfer and
+a homogeneous distribution of nuclide :math:`i` throughout the material.
+
+A more rigorous description of removal rate and its implementation can be found
+in the paper by `Hombourger
+<https://doi.org/10.1016/j.anucene.2020.107504>`_.
+
+The resulting burnup matrix can be solved with the same integration algorithms
+that are used in the absence of the transfer term.
+
+.. note::
+
+    If no ``destination_material`` is specified, nuclides that are removed
+    or fed will not be tracked afterwards.
+
+Coupling materials
+------------------
+
+To keep track of removed nuclides or to feed nuclides from one depletable material
+to another, the respective depletion equations have to be coupled. This can be
+achieved by defining one multidimensional square matrix with dimensions equal to
```

Comment:
The term "multidimensional square matrix" is confusing. If you google this, nothing turns up explicitly matching that term, and it resembles this page:

https://www.mathworks.com/help/matlab/math/multidimensional-arrays.html

which is just a tensor. And that's not what this depletion matrix is.

It's a block matrix, with diagonal blocks corresponding to depletion matrices of each material and off-diagonal blocks corresponding to inter-material coupling. We should definitely use that terminology here as it conveys what's actually happening much better.